app/javascript/dashboard/routes/dashboard/settings/teams/Index.vue

```diff
@@ -16,7 +16,7 @@
           <tbody>
             <tr v-for="item in teamsList" :key="item.id">
               <td>
-                <span class="agent-name">{{ item.name }}</span>
+                <span class="agent-name text-slate-700">{{ item.name }}</span>
```

Comment:
@iamsivin These changes are in dashboard, is it required?

Reply:
@pranavrajs I have removed color from the class `.agent-name` https://github.com/chatwoot/chatwoot/pull/4137/files#diff-6c0906a4f18055c3e96e63154041b856dd14e5fddc8d6372af39256bf8838324L45, so that is why I have added `text-slate-700` it here.

Reply:
@pranavrajs Yes, I think not required I have tried changing it, will work.

Reply:
@pranavrajs Removed the tailwind classes from dashboard https://github.com/chatwoot/chatwoot/pull/4137/commits/535e7f8989ebe9cbc361fe3f55f4fd31a0315ea1